src/analysis/analysis-model.js

```diff
@@ -25,6 +25,8 @@ var AnalysisModel = Model.extend({
     this._vis = opts.vis;
 
     this._initBinds();
+
+    this._referencedBy = {};
```

Comment:
A little comment pointing the intention would be helpful:

`// Store every layer/dataview pointing to this analysis.`